src/Tooltip/index.tsx

```diff
@@ -21,14 +22,19 @@ type Props = Pick<
  * on the element; meaning we won't show a Tooltip while the cursor is moving
  */
 export const Tooltip: React.FC<Props> = ({ children, ...props }) => {
+  const ref = React.createRef<HTMLSpanElement>();
+
   return (
-    <AbstractTooltip
-      delay={150}
-      trigger="mouseenter"
-      plugins={[mouseRest]}
-      {...props}
-    >
-      {children}
-    </AbstractTooltip>
+    <TooltipContextProvider descendsFromTooltip>
+      <AbstractTooltip
+        delay={150}
+        trigger="mouseenter"
+        plugins={[mouseRest]}
+        reference={ref.current}
+        {...props}
+      >
+        <span ref={ref}>{children}</span>
```

Comment:
I'm a little worried wrapping this is a span may break some of studio. We need this to get the mouse over events correct? 
Maybe we could do an alpha and test it out in studio before releasing :)

Reply:
This is totally a mistake that I forgot to delete. Thanks for finding this!